forte/processors/base/data_augment_processor.py

```diff
@@ -94,23 +618,40 @@ def default_configs(cls):
             - augment_entries: defines the entries the processor
             will augment. It should be a full path to the entry class.
             - other_entry_policy: a dict specifying the policies for
-            other entries.
+            other entries. It has two keys: "entry" and "policy".
 
-            If "auto_align", the span of the entry will be automatically
-            modified according to its original location. However, some
-            spans might become invalid after the augmentation, for
-            example, the tokens within a replaced sentence may disappear.
+            The values of "policy" are aligned with the values of "entry".
+            They are two aligned lists. The "entry" should be class name.
+            The "policy" specifies how to process the corresponding entries
+            after replacement.
 
-            Entries not in the dict will not be copied to the new data pack.
+            If the policy is "auto_align", the span of the entry
+            will be automatically modified according to its original location.
+            However, some spans might become invalid after the augmentation,
+            for example, the tokens within a replaced sentence may disappear.
 
-            Example: {
-                "ft.onto.base_ontology.Document": "auto_align",
-                "ft.onto.base_ontology.Sentence": "auto_align"
+            Entries not in the "entry" will not be copied to the new data pack.
+
+            Example:
+            {
+                "entry": [
+                    "ft.onto.base_ontology.Document",
+                    "ft.onto.base_ontology.Sentence"
+                ],
+                "policy": ["auto_align", "auto_align"]
             }
         """
         config = super().default_configs()
         config.update({
             'augment_entry': "ft.onto.base_ontology.Sentence",
-            'other_entry_policy': {}
+            'other_entry_policy': {
+                "entry": [],
+                "policy": [],
+            },
+            "type": "data_augmentation_op",
+            "kwargs": {
+                'data_aug_op': "",
+                'data_aug_op_config': {}
+            }
```

Comment:
```suggestion
            'other_entry_policy': {
                'kwargs': {}
            },
            'type': 'data_augmentation_op',
            'data_aug_op': '',
            'data_aug_op_config' : {
               'kwargs': {}
            }
```

Probably we should set the configuration like this? In this way, you can use arbitrary keys in `other_entry_policy`, and `data_aug_op_config`.

And, please don't mix up single quotes and double quotes in one place here.